src/cluster_legacy.c

```diff
@@ -4397,7 +4397,8 @@ static clusterMsgSendBlock *createModuleMsgBlock(int64_t module_id, uint8_t type
  *
  * If link is NULL, then the message is broadcasted to the whole cluster. */
 void clusterSendModule(clusterLink *link, uint64_t module_id, uint8_t type, const char *payload, uint32_t len) {
-    clusterMsgSendBlock *msgblock = NULL, *msgblock_light = NULL;
+    clusterMsgSendBlock *msgblock[CLUSTERMSG_HDR_NUM];
+    memset(msgblock, 0, sizeof(msgblock));
```

Comment:
Equivalent without memset?

```suggestion
    clusterMsgSendBlock *msgblock[CLUSTERMSG_HDR_NUM] = {0};
```